src/App/App.tsx

```diff
@@ -1,12 +1,29 @@
+import { useQuery } from '@apollo/client';
 import { Title } from '@mantine/core';
 import { ReactLocation, Router } from '@tanstack/react-location';
 
+import { GetAdminsIdQuery } from '../operations/queries';
+
 const location = new ReactLocation();
 
 export const App: React.FC = () => {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { data, loading, error } = useQuery(GetAdminsIdQuery);
```

Comment:
For `GetAdminsIdQuery`?

Reply:
no it was because of `loading`, it said it should be `isLoading`